packages/builder/src/components/automation/AutomationBuilder/FlowChart/TestDataModal.svelte

```diff
@@ -15,16 +15,22 @@
   let trigger = {}
   let schemaProperties = {}
 
-  // clone the trigger so we're not mutating the reference
-  $: trigger = cloneDeep(
-    $automationStore.selectedAutomation.automation.definition.trigger
-  )
+  $: trigger
+  $: schemaProperties
+  $: {
+    // clone the trigger so we're not mutating the reference
+    trigger = cloneDeep(
+      $automationStore.selectedAutomation.automation.definition.trigger
+    )
+
+    // get the outputs so we can define the fields
+    let schema = Object.entries(trigger.schema?.outputs?.properties || {})
 
-  // get the outputs so we can define the fields
-  $: schemaProperties = Object.entries(trigger?.schema?.outputs?.properties)
+    if (trigger?.event === "app:trigger") {
+      schema = Object.entries({ fields: { customType: "fields" } })
```

Comment:
It would be more explicit here to just write the actual value you need rather than use `Object.entries`:

```
[[ "fields", { customType: "fields" } ]]
```

Reply:
You're right, it's simpler, thanks